Ryujinx.HLE/HOS/Services/ServerBase.cs

```diff
@@ -266,6 +266,31 @@ private bool Process(int serverSessionHandle, ulong recvListAddr)
                     _sessions.Remove(serverSessionHandle);
                     shouldReply = false;
                 }
+                // If the type is past 0xF, we are using TIPC
+                else if ((int)request.Type > 0xF)
```

Comment:
maybe you could do that:
```suggestion
                else if ((int)request.Type > IpcMessageType.TipcCloseSession)
```